rewrite-kotlin/src/main/java/org/openrewrite/kotlin/marker/OmitBraces.java

```diff
@@ -21,8 +21,12 @@
 
 import java.util.UUID;
 
+/**
+ * Use {@link org.openrewrite.java.marker.OmitBraces} instead.
+ */
```

Comment:
Kotlin printer should be updated to respond to either the old or new marker. Kotlin parser should start using the new marker.